configure.ac

```diff
@@ -183,12 +183,20 @@ esac
 AC_SEARCH_LIBS(clock_gettime, rt)
 AC_SEARCH_LIBS(pthread_create, pthread)
 
-#
-# Prefer native kqueue(2); otherwise use libkqueue if present.
-#
-AC_CHECK_HEADER(sys/event.h, [],
-  [PKG_CHECK_MODULES(KQUEUE, libkqueue)]
+AS_IF([test -f $srcdir/libkqueue/configure.ac],
+  [AC_DEFINE(BUILD_OWN_KQUEUES, 1, [Define if building libkqueue from source])
+   ac_configure_args="--disable-libkqueue-install $ac_configure_args"
+   AC_CONFIG_SUBDIRS([libkqueue])
+   build_own_kqueues=true],
+  [build_own_kqueues=false
+   AC_CHECK_HEADER(sys/event.h, [],
+     [AC_SEARCH_LIBS([kqueue], [kqueue], [],
+       [AC_MSG_ERROR([unable to find kqueue() function])]
+     )]
```

Comment:
@seabaylea is porting my changes, here. I'm trying to remember why I changed this. I think I couldn't get M4 to parse the PKG_CHECKS_MODULES correctly when under an AS_IF - and switched to AC_SEARCH_LIBS instead.